README.md

```diff
@@ -60,9 +60,14 @@ it compatible with Zigbee networks.
     The binary with OTA header is at the same folder, ending with `z03mmc.zigbee`
 
 
-## Flashing from a custom bluetooth firmware
-1. Write transitional custom bluetooth firmware [ATC_ota_400000](./assets/ATC_ota_40000.bin) using an awesome tool from ATC_MiThermometer https://pvvx.github.io/ATC_MiThermometer/TelinkMiFlasher.html
-2. Flash z03mmc.bin firmware over transitional firmware to convert it to zigbee. Use https://devbis.github.io/telink-zigbee/ page if previous flasher stops because fo the firmware size.
+## Flashing from a custom bluetooth firmware (easy way for new devices)
+1. Open an awesome tool from ATC_MiThermometer https://pvvx.github.io/ATC_MiThermometer/TelinkMiFlasher.html
+2. Click "Connect" button and find device LYWSD03MMC
+3. After successful connecting device (Connected in logs) click "Do Activation" and wait some time
```

Comment:
Doing activation is required only for stock firmware and for ATC or other custom firmware there is no such option. It will confuse users with custom firmware.

Reply:
Fixed